src/ecalc/libraries/libecalc/common/tests/integration/snapshots/test_all_energy_usage_models/test_all_results/all_energy_usage_models_v3.json

```diff
@@ -1022,9 +1022,9 @@
                     "typ": "STREAM_DAY",
                     "unit": "MW",
                     "values": [
-                        355.37085,
-                        346.048583,
-                        358.164004,
+                        356.423481,
```

Comment:
Change is > 1 MW because there is a 0.05 power loss factor on top

Reply:
Was this pull request not about power adjustment constants? Or power loss factor? In my head thats two different things.

Guess power adjustment constant would here be 1MW?

Reply:
yes, but there is an additional power loss factor (0.05) in addition to the power adjustment constant (1 mw), which means 1 / 0.95 = 1.05... I guess

Reply:
The power adjustment constant is applied to the compressor train result (or other energy function results). The power loss factor is applied to the compressor train result (or other energy function results) before making it a consumer function result. So if the MODEL has a POWER_ADJUSTMENT_CONSTANT and the CONSUMER have a POWERLOSSFACTOR, they will accumulate in the consumer results which is what we look at in the json file here.